webpack/module-minifier-plugin/src/MinifiedIdentifier.ts

```diff
@@ -24,6 +29,7 @@ const RESERVED_KEYWORDS: string[] = [
   'double',
   'else',
   'enum',
+  'eval',
```

Comment:
"eval" isn't a reserved word in the spec. It's perfectly valid to do `var eval = 4;`